frontend/packages/console-shared/src/test-utils/utils.ts

```diff
@@ -157,3 +157,10 @@ export function searchYAML(needle: string, name: string, namespace: string, kind
   const result = execSync(`kubectl get -o yaml -n ${namespace} ${kind} ${name}`).toString();
   return result.search(needle) >= 0;
 }
+
+export const waitFor = (elem: any, target: string) => {
```

Comment:
Use proper type
```suggestion
export const waitFor = (element; any, target: string) => {
```

Reply:
There is no type definition for Element Selectors.